src/main/java/org/folio/innreach/repository/InnReachTransactionRepository.java

```diff
@@ -34,4 +35,6 @@
   @Query(name = FETCH_OPEN_BY_LOAN_ID_QUERY_NAME)
   Optional<InnReachTransaction> fetchOneByLoanId(UUID folioLoanId);
 
+  @Query(name = FETCH_BY_REQUEST_ID_QUERY_NAME)
+  Optional<InnReachTransaction> fetchOneByRequestId(UUID folioRequestId);
```

Comment:
fetchActiveByRequestId/fetchOpenByRequestId

Reply:
same for the query name

Reply:
Fixed

Reply:
Fixed